wiki/Ranking_Criteria/en.md

```diff
@@ -182,7 +182,7 @@ This category contains explicit allowance statements of concepts and rules that
 - **No two [uninherited](/wiki/Client/Beatmap_editor/Timing#uninherited-timing-point) or two [inherited timing points](/wiki/Client/Beatmap_editor/Timing#inherited-timing-point) can be placed at the same point.** Having two uninherited or two inherited timing points on top of each other will cause unintended behaviour for slider velocity and volume settings.
 - **An [inherited timing point](/wiki/Client/Beatmap_editor/Timing#inherited-timing-point) cannot be placed before the first [uninherited timing point](/wiki/Client/Beatmap_editor/Timing#uninherited-timing-point).** Without having any settings to inherit, an inherited timing point does not function properly. If you wish to alter hitsounds or slider velocities before the first uninherited timing point, it must be moved back one full measure so that inherited timing points may be used.
 - **A beatmap's first [uninherited timing point](/wiki/Client/Beatmap_editor/Timing#uninherited-timing-point) cannot be used to toggle [kiai](/wiki/Gameplay/Kiai_time).** Doing this will cause the kiai to flash before objects appear. An [inherited point](/wiki/Client/Beatmap_editor/Timing#inherited-timing-point) in the same position as the first uninherited point must be used to toggle kiai instead.
-- **Objects must be snapped to timeline ticks according to [AiMod](/wiki/Client/Beatmap_editor/AiMod).** Objects in a musical section requiring unsupported beat snap divisors (e.g. 1/11) can either:
+- **Hit objects must be snapped within less than 2 ms of any timeline tick.** [AiMod](/wiki/Client/Beatmap_editor/AiMod) will report these issues, as well as rare false positives. Objects in a musical section requiring unsupported beat snap divisors (e.g. 1/11) can either:
```

Comment:
```suggestion
- **Hit objects must be snapped within less than 2 ms of any timeline tick.** [AiMod](/wiki/Client/Beatmap_editor/AiMod) will report these issues, as well as rare false positives. False positives mainly occur on slider ends and reverses, and should be verified manually or with other tools. Objects in a musical section requiring unsupported beat snap divisors (e.g. 1/11) can either:
```

Reply:
This seems like a roundabout way of suggesting:

```suggestion
- **Hit objects must be snapped within less than 2 ms of any timeline tick.** Use [MV](https://github.com/Naxesss/MapsetVerifier) or the lazer client to detect these issues accurately. Objects in a musical section requiring unsupported beat snap divisors (e.g. 1/11) can either:
```

Which is more concise and skips the hassle with AiMod entirely. AiMod may be fine for detecting the majority of these cases, but with it being unmaintained and outdated, I don't think it's something we should promote in the RC at all.

Reply:
^ I agree that this would work better